setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
+max-line-length = 99
+
+[pylint.MASTER]
```

Comment:
I saw also that `pylint` is pointing our TODOs in the code. I think we should skip that as well, as in all of the repositories we have plenty of them and it can only make the result less readable:
```cfg
[pylint.MISCELLANEOUS]
; http://pylint.pycqa.org/en/latest/technical_reference/features.html#miscellaneous-checker-options
notes = FIXME
```

I left `FIXME` as a possibility to mark something that should be fixed within one pull request. Does it make sense to you? 🙂